### PR TITLE
Istio remote installation updates

### PR DESCRIPTION
**Pull request by ymesika**
Depends on https://github.com/istio/istio/pull/11307

Docs of installing VPN / Split Horizon EDS have been updated to fit the new way of installing the `istio-remote` from a values file rather than having its own chart.

### CODE REVIEW

File: content/docs/setup/kubernetes/multicluster-install/vpn/index.md [hunk 1]
```diff
@@ -167,7 +168,7 @@ perform a manual sidecar injection refer to the [manual sidecar example](#manual
 1. Install the Helm chart for the `istio-remote` with the following command:
 
     {{< text bash >}}
-    $ helm install install/kubernetes/helm/istio-remote --name istio-remote  --namespace istio-system --set global.remotePilotAddress=${PILOT_POD_IP} --set global.remotePolicyAddress=${POLICY_POD_IP} --set global.remoteTelemetryAddress=${TELEMETRY_POD_IP} --set global.proxy.envoyStatsd.enabled=true --set global.proxy.envoyStatsd.host=${STATSD_POD_IP} --set global.remoteZipkinAddress=${ZIPKIN_POD_IP}
+    $ helm install install/kubernetes/helm/istio --name istio-remote --namespace istio-system --values install/kubernetes/helm/istio/values-istio-remote.yaml --set global.remotePilotAddress=${PILOT_POD_IP} --set global.remotePolicyAddress=${POLICY_POD_IP} --set global.remoteTelemetryAddress=${TELEMETRY_POD_IP} --set global.proxy.envoyStatsd.enabled=true --set global.proxy.envoyStatsd.host=${STATSD_POD_IP} --set global.remoteZipkinAddress=${ZIPKIN_POD_IP}
```
Comment (sdake) [inline content/docs/setup/kubernetes/multicluster-install/vpn/index.md]: statsd is long gone.  Please remove from the docs.

Zipkin never worked with istio-remote.  Please remove from the docs.

Not sure on the need for policy or telemetry.  Can you verify?

File: content/docs/setup/kubernetes/multicluster-install/vpn/index.md [hunk 2]
```diff
@@ -353,7 +354,7 @@ Before you begin, set the endpoint IP environment variables as described in the
    control plane service endpoints:
 
     {{< text bash >}}
-    $ helm template install/kubernetes/helm/istio-remote --namespace istio-system --name istio-remote --set global.remotePilotAddress=${PILOT_POD_IP} --set global.remotePolicyAddress=${POLICY_POD_IP} --set global.remoteTelemetryAddress=${TELEMETRY_POD_IP} --set global.proxy.envoyStatsd.enabled=true --set global.proxy.envoyStatsd.host=${STATSD_POD_IP} --set global.remoteZipkinAddress=${ZIPKIN_POD_IP} --set sidecarInjectorWebhook.enabled=false > $HOME/istio-remote_noautoinj.yaml
+    $ helm template install/kubernetes/helm/istio --namespace istio-system --name istio-remote --values install/kubernetes/helm/istio/values-istio-remote.yaml --set global.remotePilotAddress=${PILOT_POD_IP} --set global.remotePolicyAddress=${POLICY_POD_IP} --set global.remoteTelemetryAddress=${TELEMETRY_POD_IP} --set global.proxy.envoyStatsd.enabled=true --set global.proxy.envoyStatsd.host=${STATSD_POD_IP} --set global.remoteZipkinAddress=${ZIPKIN_POD_IP} --set sidecarInjectorWebhook.enabled=false > $HOME/istio-remote_noautoinj.yaml
```
Comment (sdake) [inline content/docs/setup/kubernetes/multicluster-install/vpn/index.md]: same comments applies as above.

**Comment by ymesika** [comment]
Dependency https://github.com/istio/istio/pull/11307 is has been merged.
Please merge this one too to keep doc updated.

**Comment by geeknoid** [comment]
@sdake Is this good to go?

**Pull request merged**